### DISCUSSION TITLE
How do i adfress all characters in the hex tables?

### DISCUSSION THREAD

**Discussion by natabma** [olikraus/u8g2#2424] [Q&A]
Hi all,

tables such as these (https://github.com/olikraus/u8g2/wiki/fntgrpunifont) have more than 10 colums. However, the rows increase by 10.

How do i adress for example the 😍 emoji in the u8g2_font_unifont_t_emoticons table? What is the hex code?

![image](https://github.com/olikraus/u8g2/assets/77214481/f37b2beb-4097-4b0d-a30d-99230dd9a173)

**Comment by olikraus** [comment]
The tables include exactly 16 columns. The first number is the decimal number of the first column and the second number is the hex version of the same decimal number.

If you want to print a glyph in your code with the Arduino print() function or with any other function you need to add the glyph code into your string. This can be done with the \x escape sequence, which supports hex codes (thats why the second number is the hex number).

So your red circled glyph is at position 13, which is "c" in hex. So you need to add "c" to the hex number of the same row, which is "20", so we get "2c".

So you end up with "u8g2.print("\x2c");

Of course you can also draw the glyph directly with 
```
u8g2.drawGlyph(0, 10, 0x2c);
```

Note that the syntax differs between the "in-string" escape number (\x prefix) and the plan C number (0x prefix)
